### PR TITLE
First iteration

### PR DESCRIPTION
**Pull request by johannesu**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+    <dependencies>
+
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-web</artifactId>
+            <version>${springVersion}</version>
+        </dependency>
+
+        <dependency>
```
Comment (nealstewart) [inline pom.xml]: It seems strange that you have Jetty configured both in your Maven dependencies and in your Dockerfile

File: src/main/java/com/tradeshift/elektra/johannes/MessageDTO.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    public MessageDTO(String message) {
+        this.message = new Content(message);
+    }
+
+    public Content getMessage() {
+        return message;
+    }
+}
+
+class Content {
```
Comment (nealstewart) [inline src/main/java/com/tradeshift/elektra/johannes/MessageDTO.java]: Why the extra Content abstraction?

**Comment by nealstewart** [comment]
I'd clean up your commit history a little bit.

I usually squash the whole thing into a single commit before merging to master. This varies, but ideally your git history should only contain relevant commits to the history of the repository.

For instance, there's no need for the project to remember you removed an unused plugin, since it was never used in master.

`git fetch; git rebase -i origin/master` will take you through an interactive merge process.

Those are all my comments.

File: src/main/java/com/tradeshift/elektra/johannes/MessageDTO.java (hunk 2 above)
Comment (johannesu) [inline src/main/java/com/tradeshift/elektra/johannes/MessageDTO.java]: To get proper formatting on the JSON response.

**Comment by nealstewart** [comment]
:+1:

**Pull request merged**